bsdetector/bias.py

```diff
@@ -612,25 +636,3 @@ def enumerate_sentences(fpath='input_text'):
             yield(biasq, statement)
         else:
             print('-- Statement is too short: {}'.format(statement))
-
```

Comment:
can this code get worked into a pytest?

Reply:
Yes sure, I can figure it out.

Reply:
Yes sure, I can figure it out.

Reply:
I have added test for json print output function, that is the same used by default command-line entry point. As Python 2 execution time is much slower compared to Python 3, test on full article is timing out on Travis CI (> 10 mins). I can change to test to compare a shorter article.

Reply:
Probably because of this https://stackoverflow.com/questions/15639779/why-does-multiprocessing-use-only-a-single-core-after-i-import-numpy/15641148#15641148